webroot/js/components/chat/chat-input.js

```diff
@@ -234,6 +235,9 @@ export default class ChatInput extends Component {
 
   handleMessageInputKeyup(event) {
     const { key } = event;
+    const { inputHTML } = this.state;
```

Comment:
These are no longer needed. 
The setState() at the end is therefore also not needed.
Instead of counting chars here, we'll let the downstream component update on `handleContentEditableChange` handle the updates of the charsLeft in state.